### PR TITLE
fix: MD033/no-inline-html

### PR DESCRIPTION
**Pull request by nschonni**
Escape elements swallowed as HTML in markdown.
Stripped out span and custom styles that get removed on publish.
Replace i/b elements

### CODE REVIEW

File: docs/cross-platform/macios/unified/updating-mac-apps.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ Add the following lines of code after the `<AssemblyName>` element:
 
 Example:
 
-![](updating-mac-apps-images/csproj3.png "Add these lines of code after the <AssemblyName> element")
+![](updating-mac-apps-images/csproj3.png "Add these lines of code after the \<AssemblyName> element")
```
Comment (conceptdev) [inline docs/cross-platform/macios/unified/updating-mac-apps.md]: Escaping here is not required, the alt text renders correctly on the current published site:

Screen Shot 2019-07-14 at 1 28 30 PM

File: docs/index.md [hunk 2]
```diff
@@ -9,6 +9,7 @@ ms.date: 02/12/2019
 featureFlags:
 - clicktale
 ---
+<!-- markdownlint-disable MD033 -->
```
Comment (conceptdev) [inline docs/index.md:12]: nice touch

**Review by conceptdev — CHANGES_REQUESTED** [review]
One of the changes isn't required (in image alt text). Thanks for the attention to detail on the rest!

File: docs/cross-platform/macios/unified/updating-mac-apps.md (hunk 1 above)
Comment (nschonni) [reply docs/cross-platform/macios/unified/updating-mac-apps.md]: Ah, I found it was before in the alt text. This actually isn't in the alt text, but the title. Should it be moved to the alt text?

Comment (conceptdev) [reply docs/cross-platform/macios/unified/updating-mac-apps.md]: I always get confused which maps to which in Markdown. No need to move it, I think we'll be doing an accessibility review soon to determine the best handling of title _and_ alt across the entire docset.

Comment (nschonni) [reply docs/cross-platform/macios/unified/updating-mac-apps.md]: Inside the bracket is alt text. The trailing string is title. Title doesn't work for Screen Readers or mobile.

**Review by conceptdev — APPROVED** [review]

**Pull request merged**